lms/djangoapps/teams/static/teams/js/teams_tab_factory.js

```diff
@@ -1,13 +1,31 @@
 ;(function (define) {
     'use strict';
-
-    define(['jquery', 'underscore', 'backbone', 'teams/js/views/teams_tab'],
-        function ($, _, Backbone, TeamsTabView) {
+    define([
+        'jquery',
+        'underscore',
+        'backbone',
+        'gettext',
+        'teams/js/views/teams_tab',
+        'teams/js/views/team_utils'
+    ], function ($, _, Backbone, gettext, TeamsTabView, TeamUtils) {
             return function (options) {
                 var teamsTab = new TeamsTabView({
                     el: $('.teams-content'),
                     context: options
                 });
+                $(document).ajaxError(function (event, xhr) {
+                    var message;
+                    if (xhr.status === 401) {
+                        message = gettext("We couldn't complete your request. Please reload the page and try again.");
+                    }
+                    else if (xhr.status === 500) {
+                        message = gettext("We had some trouble processing your request. Please try again.");
+                    }
```

Comment:
@catong That sounds good to me -- I think the important thing is that the user be prompted to reload the page in the case of a 401, since that occurs when they aren't logged in.